src/operations.rs

```diff
@@ -86,7 +88,7 @@ pub fn install_version(
 
     std::fs::create_dir_all(target_path.parent().unwrap())?;
 
-    eprintln!("Installing Julia {} ({}).", version, platform);
+    eprintln!("{} Julia {} ({}).", style("Installing").green().bold(), version, platform);
```

Comment:
We might check whether that `console` package is smart enough to detect whether we are connected to a tty or not, and then only style things when we are.

Reply:
Looking at https://docs.rs/console/0.14.1/console/fn.colors_enabled.html, I think this should be disabled when the process is piped but good to confirm.

Reply:
I also looked through their issues, and I think this should be ok. We use the `atty` package to detect that, they also did at some point, then removed it because they said they had all that logic vendored, so clearly they are aware of this.